medusa/clients/torrent/utorrent_client.py

```diff
@@ -54,17 +56,63 @@ def _get_auth(self):
             return self.auth
 
     def _add_torrent_uri(self, result):
-        return self._request(params={
+            ## Set proper subfoler as destination for uTorrent 
+            series_id = result.series.series_id
+            series_name = result.series.name
+ 
+			# Get the sub-folder the user has assigned to that series
```

Comment:
Indent correctly, comments should be on the same line with code.

Reply:
Done. Having some issues with my editor...